### PR TITLE
main to production

### PR DESCRIPTION
**Pull request by SimonOConnor28**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
```
Comment (petrungaro) [inline src/App.js:15]: Very tidy App.js, I like that everything it component-ized

File: src/Monday.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            setUserItemMonday("");
+          };
+
+          const handleRemoveClick = (itemKey) => {
+            const database = getDatabase(firebase);
+            const dbRef = ref(database, `/monday/${itemKey}`);
+            remove(dbRef);
+          };
+          return (
+            <div className="tuesBackground sectionSpace">
```
Comment (petrungaro) [inline src/Monday.js:48]: I noticed you have a class tuesBackground in several areas (other than the Tuesday component) - is this intentional or maybe you meant to change it?

File: src/Friday.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import firebase from "./firebase";
```
Comment (petrungaro) [inline src/Friday.js:1]: This is just a general comment, no action needed, and I'm not sure if it makes sense. It seems like all the Day components use the same functions (handleInputChange, handleClick, handleRemoveClick). Maybe there would be a way to define them in App and pass them to each individual component, so you wouldn't have to write them multiple times ?

**Review by petrungaro — COMMENTED** [review]
Looks really good!

File: src/App.css [hunk 4]
```diff
@@ -1,38 +1,175 @@
...
+
+input::placeholder {
+  overflow: visible;
+}
+
+.listFlex {
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
```
Comment (jhshim85) [inline src/App.css:89]: You could remove this line.
by default, flex-direction is 'row'

File: src/App.js [hunk 5]
```diff
@@ -1,23 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
-
+import Header from './Header';
+import Monday from './Monday';
```
Comment (jhshim85) [inline src/App.js:3]: love the idea of using each day as an individual component

**Review by jhshim85 — COMMENTED** [review]
great code and styling!
just love how you managed to put each day's data into individual component!

**Pull request merged**